ipfix/test_ipfix.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+
+# Unit tests for IPFIX plugin
+#
+# How to run
+# ----------
+# Since the plugin is out of the VPP tree, running the tests has extra steps.
+#
+# To run tests in general in the VPP working dir:
+#   $ make test
+#
+# Now to run for this plugin:
+#   $ make test EXTERN_TESTS=<path-to-plugin>/ipfix
+#
+# To only run this particular test:
+#   $ make test TEST=test_ipfix EXTERN_TESTS=<path-to-plugin>/ipfix \
+#               EXTERN_PLUGIN=/usr/lib/vpp_plugins
+#
+# To help debug the tests, you can supply a verbosity flag, e.g., "V=1"
```

Comment:
I think this probably should go in the module level docstring. This is more of an aesthetic change but it's more pythonic to do that in my opinion.

Something like this:
```python
#!/usr/bin/env python
"""
Unit tests for IPFIX plugin

...

To help debug the tests, you can supply a verbosity flag e.g., V=1
"""
```
More info here: https://www.python.org/dev/peps/pep-0257/